cmd/terramate/e2etests/run_test.go

```diff
@@ -78,6 +78,27 @@ frita
 `,
 			},
 		},
+		{
+			name: "independent stacks inside other stacks gives consistent ordering (lexicographic by path)",
```

Comment:
I imagine this will change on the next PR when we introduce the ordering changes (nice to do in multiple steps)

Reply:
yep